packages/ui/src/modal/SelectPaymentToken.tsx

```diff
@@ -23,74 +23,76 @@ export const SelectPaymentToken: FC<Props> = ({
 }) => {
   return (
     <Flex direction="column" css={{ width: '100%', gap: '$1', px: '$3' }}>
-      {paymentTokens?.map((paymentToken) => {
-        const isSelectedCurrency =
-          currency?.address.toLowerCase() === paymentToken?.address
-        const formattedBalance = formatUnits(
-          BigInt(paymentToken?.balance || 0),
-          paymentToken?.decimals || 18
-        )
+      {paymentTokens
+        ?.sort((a, b) => Number(a.balance) - Number(b.balance))
```

Comment:
I think in the end we actually decided it'd be best to sort by total price in the currency. So should be 

```
{paymentTokens
        ?.sort((a, b) => Number(a.currencyTotalFormatted) - Number(b.currencyTotalFormatted))
```